components/live/header.tsx

```diff
@@ -26,8 +28,13 @@ export const LiveHeader = (props: LiveHeaderprops) => {
               "flex flex-row items-center max-md:justify-center"
             )}
           ></div>
+          <h1
+            dangerouslySetInnerHTML={{
+              __html: sanitiseXSS(title, spanWhitelist) || "",
+            }}
+          ></h1>
           <div className="mb-2 pb-1 pt-3 text-5xl child-h1:text-5xl child-h1:font-semibold child-p:text-xl descendant-strong:font-semibold descendant-strong:text-sswRed">
```

Comment:
Can you remove the classes that still make it bold? Also copy all the child-h1 stuff to the className of the h1 above (without the child-h1: as a prefix)